driver-dse/src/main/java/com/datastax/driver/graph/Edge.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.graph;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+/**
+ * A default representation of an Edge in DSE Graph.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonDeserialize(using = DefaultEdgeDeserializer.class)
+public class Edge {
```

Comment:
+1 on this.

> One hurdle is that gremlin-driver compiles to Java 8, so we'd have to raise the requirement on our dse module.

We already do this for driver-extras so it's technically easy; however that might put some DSE users out of the game.